.github/workflows/smoke-test-all.yml

```diff
@@ -44,12 +43,6 @@ jobs:
         working-directory: ${{ env.WORKING_DIRECTORY }}
         run: npm run build
 
-      - name: Core - Publish (dry-run)
-        working-directory: ${{ env.WORKING_DIRECTORY }}
-        run: npm publish --tag latest --dry-run
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-
   release-angular:
```

Comment:
Maybe to change name to "build angular" ?

Reply:
yea true, probably on all steps